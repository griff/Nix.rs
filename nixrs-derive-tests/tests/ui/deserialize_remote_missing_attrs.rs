use nixrs_derive::nix_deserialize_remote;

pub struct Test;

nix_deserialize_remote!(Test);

pub fn main() {}
