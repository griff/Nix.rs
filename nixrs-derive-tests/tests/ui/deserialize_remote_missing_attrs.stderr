error: Missing from_str, from_store_dir_str, from or try_from attribute
 --> tests/ui/deserialize_remote_missing_attrs.rs
  |
  | nix_deserialize_remote!(Test);
  |                         ^^^^
